src/mail/view/MailViewerViewModel.ts

```diff
@@ -113,6 +114,9 @@ export class MailViewerViewModel {
 	private readonly loadingState = new LoadingStateTracker()
 
 	private renderIsDelayed: boolean = true
+	// We render the mailbody to a shadowdom, which doesn't inherit styles from the global stylesheet
+	// So we just copy it and put it in the shadowdom
+	private styleSheet: Node = styles.getStyleSheetElement("main")
```

Comment:
let's see if we need to continue doing it but it doesn't belong in viewModel anyway IMO